src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Tables/Instance.sql

```diff
@@ -45,53 +48,22 @@ INCLUDE
 )
 WITH (DATA_COMPRESSION = PAGE)
 
--- Used in WADO and QIDO, putting PartitionKey last allows us to query across partitions in the future.
-CREATE NONCLUSTERED INDEX IX_Instance_StudyInstanceUid_Status_PartitionKey on dbo.Instance
+-- Used in WADO
+CREATE NONCLUSTERED INDEX IX_Instance_PartitionKey_Status_StudyInstanceUid_SeriesInstanceUid_SopInstanceUid on dbo.Instance
 (
-    StudyInstanceUid,
+    PartitionKey,
     Status,
-    PartitionKey    
-)
-INCLUDE
-(
-    SeriesInstanceUid,
-    SopInstanceUid,
-    Watermark,
-    TransferSyntaxUid
-)
-WITH (DATA_COMPRESSION = PAGE)
-
--- Used in WADO and QIDO, putting PartitionKey last allows us to query across partitions in the future.
-CREATE NONCLUSTERED INDEX IX_Instance_StudyInstanceUid_SeriesInstanceUid_Status_PartitionKey on dbo.Instance
-(
     StudyInstanceUid,
     SeriesInstanceUid,
-    Status,
-    PartitionKey    
+    SopInstanceUid
 )
 INCLUDE
 (
-    SopInstanceUid,
```

Comment:
Do you think adding HasFrameData helps here for GetInstance sproc?

Reply:
Do you think adding HasFrameData helps here for GetInstance sproc?

Reply:
Thanks. Just checked, it does index seek on this + keylookup which is expected. With "SET STATISTICS TIME ON" it took 50ms for 3 rows. When I include it does not do Keylookup as expected and took 46ms. 

Since it is just a bit column the added size is not a lot, so I will add it